### PR TITLE
chore(build): remove the auto-generated cpp code by thrift compiler 

### PR DESCRIPTION
**Pull request by shenxingwuying**
Need not save the files generated by thrift in the repo

issue: https://github.com/apache/incubator-pegasus/issues/1040

### CODE REVIEW

File: rdsn/thirdparty/CMakeLists.txt [hunk 1]
```diff
@@ -409,3 +409,13 @@ ExternalProject_Add(http-parser
         INSTALL_COMMAND cp -R http_parser.h ${TP_OUTPUT}/include/nodejs/http_parser.h
         BUILD_IN_SOURCE 1
         )
+
+# ExternalProject_Add(python
+#         URL ${OSS_URL_PREFIX}/Python-2.7.18.tgz
+#         https://www.python.org/ftp/python/2.7.18/Python-2.7.18.tgz
```
Comment (acelyc111) [inline rdsn/thirdparty/CMakeLists.txt:415]: python3 is installed on each Dockerfile, no need to installed again. See
https://github.com/apache/incubator-pegasus/blob/master/docker/pegasus-build-env/centos7/Dockerfile#L28

Comment (shenxingwuying) [reply rdsn/thirdparty/CMakeLists.txt]: python3 . ok .
I'll try again.

**Comment by acelyc111** [comment]
please rebase the latest master branch to resolve conflicts, thanks

**Review by acelyc111 — APPROVED** [review]
Thanks!

**Review by levy5307 — APPROVED** [review]

**Pull request merged**